### PR TITLE
Write README.md file

### PR DESCRIPTION
**Pull request by Lookingfor00**
Write the README.md file content

### CODE REVIEW

**Comment by Lookingfor00** [comment]
Linter causes error again: Unexpected empty source   no-empty-source

I will delete the content of the index.css

**Comment by Lookingfor00** [comment]
I finally decided to create another branch add-header to write index.html and index.css content before trying to merge this branch.
Because linter won't let me merge until a write the content of index.html and index.css but I can't write them in this branch.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Microverse-WorkFlow1
-Set up a "Hello Microverse" project following a specific work flow
+Set up a "Hello Microverse" project following a specific Microverse work flow
+
```
Comment (christianbiring1) [inline README.md:5]: - [ ] Kindly make sure your project has a descriptive `title` and `detailed description` saying what the project is about. :smile: 
Hint: You use the ones provided in the project requirements

**Review by christianbiring1 — COMMENTED** [review]
Hi @Trast00 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS : REQUEST CHANGES :recycle: 
almost

## Highlights
- [ ] Linters are set even though faling
- [ ] test file is ignored by git

## Required Changes ♻️
- [ ] Kindly make sure you use the correct HTML boilerplate to generate the initial setup of your project.
Hint: To do so you can hit `! and then the tab button`
- [ ] Kindly make sure your HMTL file is linked with your CSS file. You can check [here](https://www.youtube.com/results?search_query=how+to+link+html+to+css)
As you can see no color is applied to your h1
image
- [ ] Kindly make sure all the linters checks are green :heavy_check_mark: 
Hint: After you run the command stated in the `linter.yml` file `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x` make sure you fix them with `npx stylelint "**/*.{css,scss}" --fix`
_Check the comments under the review._

## Optional suggestions
- [ ] Also please, On every project you will be doing later in the program, make sure you have opened only `ONE` Pull Request that introduces the changes you have made in the project feature.


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**